addons/mail/static/src/models/rtc_data_channel/rtc_data_channel.js

```diff
@@ -0,0 +1,39 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '../../model/model_field_command';
+
+registerModel({
+    name: 'RtcDataChannel',
+    identifyingFields: ['rtcSession'],
+    lifecycleHooks: {
+        _willDelete() {
+            this.dataChannel.close();
+        },
+    },
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeRtc() {
+            return this.messaging && this.messaging.rtc ? replace(this.messaging.rtc) : clear();
+        },
```

Comment:
This is just a related?
And why do we need to define the field specifically?

Reply:
Was done so that we could delete all `RtcDataChannel`s in specific flow of `Rtc`.
I removed this and instead just loop on all `RtcDataChannel` and delete them, without relying on a relational field.